support-workers/src/test/scala/com/gu/support/workers/integration/DigitalSubscriptionGiftRedemptionIntegrationSpec.scala

```diff
@@ -25,6 +24,7 @@ class DigitalSubscriptionGiftRedemptionIntegrationSpec extends AsyncLambdaSpec w
   val createSubRequestId = UUID.randomUUID()
   val redeemSubRequestId = UUID.randomUUID()
   val giftCode = new GiftCodeGeneratorService().generateCode(Annual)
+  SafeLogger.info(s"Gift code = ${giftCode}")
```

Comment:
didn't realise that anything in the test class will happen right at the startt  of the test run
![image](https://user-images.githubusercontent.com/7304387/94810693-43d30480-03ec-11eb-95e2-7c1b722602d7.png)
might be better to log it in the first test instead @rupertbates ?